### PR TITLE
[DOCS] Added Documentation of AudioStreamPlayer

### PR DESCRIPTION
**Pull request by Grosskopf**
and the trio is done :)

### CODE REVIEW

File: doc/classes/AudioStreamPlayer.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamPlayer" inherits="Node" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays back audio.
 	</brief_description>
 	<description>
+		Plays background audio without dampening according to distance
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer.xml]: I don't think you need to go into what it *doesn't* do.  I don't think it's clear what "distance" refers to anyway.

Also, the sentence is missing a period.

File: doc/classes/AudioStreamPlayer.xml [hunk 2]
```diff
@@ -111,26 +115,33 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer.xml]: Not at the start of the game - it plays when the node enters the scene tree.  Also add default value.

File: doc/classes/AudioStreamPlayer.xml [hunk 3]
```diff
@@ -111,26 +115,33 @@
...
+			The [AudioStream] object to be played.
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">
+			Volume of sound, in dB.
 		</member>
 	</members>
 	<signals>
 		<signal name="finished">
 			<description>
+				Fires when the audio stops playing.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer.xml]: Use "emitted", not "fires".

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/AudioStreamPlayer.xml (hunk 2 above)
Comment (Noshyaar) [reply doc/classes/AudioStreamPlayer.xml]: Please remove single quotes inside [code] tag too.

File: doc/classes/AudioStreamPlayer.xml [hunk 4]
```diff
@@ -111,26 +115,33 @@
...
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
 		</member>
 		<member name="bus" type="String" setter="set_bus" getter="get_bus">
+			Bus on which this audio is playing.
 		</member>
 		<member name="mix_target" type="int" setter="set_mix_target" getter="get_mix_target" enum="AudioStreamPlayer.MixTarget">
 		</member>
 		<member name="playing" type="bool" setter="_set_playing" getter="is_playing">
+			If [code]'true'[/code], audio is playing.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer.xml]: Remove the quotes around 'true'.

File: doc/classes/AudioStreamPlayer.xml (hunk 3 above)
Comment (Noshyaar) [reply doc/classes/AudioStreamPlayer.xml]: I also see both are used in other files. Should we use only 'emitted'? I'll add that to my checklist.

File: doc/classes/AudioStreamPlayer.xml [hunk 5]
```diff
@@ -65,6 +68,7 @@
 			<argument index="0" name="to_pos" type="float">
 			</argument>
 			<description>
+				Sets the position from wich audio will be played (in seconds).
```
Comment (Noshyaar) [inline doc/classes/AudioStreamPlayer.xml]: `which`

File: doc/classes/AudioStreamPlayer.xml (hunk 3 above)
Comment (cbscribe) [reply doc/classes/AudioStreamPlayer.xml]: I've been removing them when I find them. Sticking with engine terminology - signals use `emit` not `fire`.

**Pull request merged**